client/src/components/QuestionsAndAnswers/QuestionAnswer.jsx

```diff
@@ -14,6 +14,10 @@ const QABlock = styled.div`
     padding: 20px;
 `;
 
+const BoldSpan = styled.span`
+    font-weight: 600;
+`;
+
 const Question = styled.div`
```

Comment:
This div has the same properties as BoldSpan, could you consolidate them into one? Or if they will later have additional unique stylings then having two divs makes sense

Reply:
good suggestion! i didn't even notice that they were the same, I'm planning to add more styling later so these will be revised later.